pandas/tests/groupby/test_apply.py

```diff
@@ -1011,25 +1011,6 @@ def test_result_order_group_keys_false():
     tm.assert_frame_equal(result, expected)
 
 
-def test_groupby_apply_group_keys_warns():
-    df = DataFrame({"A": [0, 1, 1], "B": [1, 2, 3]})
-    msg = "Not prepending group keys to the result index"
-    with tm.assert_produces_warning(FutureWarning, match=msg):
-        result = df.groupby("A").apply(lambda x: x)
```

Comment:
Is it worth keeping these tests with `group_keys=False` for testing coverage, or would that be redundant with existing tests?

Reply:
As part of the original deprecation, many tests that use apply now have `group_keys=False`. Throughout the tests, there are 88 lines of code where we specify `group_keys=False`. I feel pretty good about our test coverage without this, but no real objection if you want it kept.

Reply:
Sounds good. I'm fine with removing these tests then.